### PR TITLE
build(distro-wildfly) Use DMP to produce WF10 docker image using latest code.

### PR DESCRIPTION
**Pull request by msavy**
CC @jcechace

### CODE REVIEW

File: distro/wildfly10/pom.xml [hunk 1]
```diff
@@ -144,32 +148,50 @@
...
+                    <name>apiman/on-wildfly10:master</name>
+                    <alias>wildfly-master</alias>
+                    <build>
+                      <dockerFileDir>${project.basedir}/src/main/docker</dockerFileDir>
+                      <assembly>
+                        <inline>
+                          <fileSets>
+                            <fileSet>
+                              <includes>
+                                <include>target/apiman-distro-wildfly10-1.3.2-SNAPSHOT-overlay.zip</include>
```
Comment (msavy) [inline distro/wildfly10/pom.xml]: Whoops. Fix literal reference.

File: distro/wildfly10/pom.xml [hunk 2]
```diff
@@ -144,32 +148,50 @@
...
+                  <id>docker-build</id>
+                  <goals>
+                    <goal>build</goal>
+                  </goals>
+                </execution>
+              </executions>
+              <configuration>
+                <images>
+                  <image>
+                    <name>apiman/on-wildfly10:master</name>
```
Comment (msavy) [inline distro/wildfly10/pom.xml]: Use variable?

**Comment by jcechace** [comment]
How will this compare to https://github.com/apiman/apiman-docker?  

For a fast quickstart this plugin approach seems better, whereas the standalone repository could be used for more sophisticated (production-ready if you will) images?

**Comment by msavy** [comment]
> For a fast quickstart this plugin approach seems better, whereas the standalone repository could be used for more sophisticated (production-ready if you will) images?

Yes, that was my intention (and it seems that this was the most practical way of producing a master build).  I'm still open minded longer term if someone comes up with a better idea.

**Pull request merged**